Source/Examples/ExampleLibrary/ExampleLibrary.csproj

```diff
@@ -1,27 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <TargetFrameworks>netstandard1.0;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462;net6.0;net6.0-windows</TargetFrameworks>
+    <TargetFrameworks>net462;net6.0;net6.0-windows</TargetFrameworks>
```

Comment:
Extra line

Reply:
Okay wow, the Git diff algorithm was not helpful this time.